packages/block-library/src/group/edit.js

```diff
@@ -43,6 +48,13 @@ function GroupEdit( {
 						},
 					] }
 				/>
+				<PanelBody title={ __( 'Spacing' ) }>
+					<DimensionControl
+						label={ 'Padding' }
+						icon={ 'desktop' }
+						iconLabel={ 'All Devices' }
```

Comment:
This string is not translatable: should use `__()`

Reply:
Whilst you are entirely correct, this code was purely for testing purposes (I did note this in description but it was easy to miss). It has been removed as this is now close to merge.